apps/api-portal/cloudbuild.yaml

```diff
@@ -8,13 +8,14 @@ steps:
 
   - id: 'install-yarn'
     waitFor: ['restore_cache']
-    name: gcr.io/cloud-builders/yarn:latest
-    entrypoint: bash
+    name: node:16-alpine
```

Comment:
Because the pre-built yarn image from gcr.io runs on node 14.17.1 that is not compatible with `eslint-import-resolver-typescript@3.5.2`, thus failing to install all the deps
```
Step #1 - "install-yarn": error eslint-import-resolver-typescript@3.5.2: The engine "node" is incompatible with this module. Expected version "^14.18.0 || >=16.0.0". Got "14.17.1"
Step #1 - "install-yarn": info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Step #1 - "install-yarn": error Found incompatible module.
```